### PR TITLE
add pact -s

### PR DESCRIPTION
**Pull request by rsoeldner**
PR checklist:

* [ ] Test coverage for the proposed changes
* [ ] PR description contains example output from repl interaction or a snippet from unit test output
* [ ] (If Relevant) Documentation has been (manually) updated at https://docs.kadena.io/pact

Additionally, please justify why you should or should not do the following:

* [ ] Benchmark regressions
* [x] Confirm replay/back compat (Ignore until core release)
* [x] (For Kadena engineers) Run integration-tests against a Chainweb built with this version of Pact (Ignore until core release)

### CODE REVIEW

File: repl/Main.hs [hunk 1]
```diff
@@ -113,6 +117,21 @@ main = O.execParser argParser >>= \case
           Just s -> runScript s dbg
           Nothing -> runScript fp dbg
       | otherwise -> runScript fp dbg
+    OServer configPath -> Y.decodeFileEither configPath >>= \case
+      Left perr -> putStrLn $ Y.prettyPrintParseException perr
+      Right config -> runServer config
+      --do
```
Comment (jmcardon) [inline repl/Main.hs:123]: Clean up

File: .vscode/settings.json [hunk 2]
```diff
@@ -0,0 +1,7 @@
+{
```
Comment (rsoeldner) [inline .vscode/settings.json:1]: This seems like an accident.

File: pact-request-api/Pact/Core/Command/Server/Config.hs [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+    <*> o JD..:? "gasLimit"
+
+usage :: String
+usage = unlines
+  [ "Config file is YAML format with the following properties:"
+  , "port       - HTTP server port"
+  , "persistDir - Directory for database files."
+  , "             If omitted, runs in-memory only."
+  , "logDir     - Directory for HTTP logs, defaults to no log dir"
+  , "pragmas    - SQLite pragmas to use with persistence DBs"
```
Comment (rsoeldner) [inline pact-request-api/Pact/Core/Command/Server/Config.hs:40]:
```suggestion
```

File: pact-request-api/Pact/Core/Command/Server/Config.hs [hunk 4]
```diff
@@ -0,0 +1,56 @@
...
+  , "entity     - Entity name for simulating privacy, defaults to \"entity\""
```
Comment (rsoeldner) [inline pact-request-api/Pact/Core/Command/Server/Config.hs:41]:
```suggestion
```

File: pact-request-api/Pact/Core/Command/Server/Config.hs (hunk 3 above)
Comment (rsoeldner) [inline pact-request-api/Pact/Core/Command/Server/Config.hs:40]: Does not match the current config type.

**Review by jmcardon — APPROVED** [review]

**Pull request merged**